public/fullSchema.json

```diff
@@ -7,6 +7,10 @@
     "contact": {
       "name": "ChilliPharm",
       "url": "https://www.chillipharm.com"
+    },
+    "license": {
```

Comment:
You can remove the license - it's just a warning in the lint so no need to worry about it - just when it's an error do we act on it